pages/about.html

```diff
@@ -7,43 +7,42 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>&hearts;shell-bee&hearts;</title>
 </head>
+
 <body>
   <header>
     <h1>&hearts;about&hearts;</h1>
-    <nav>
-      <div class="dropdown">
-          <span>navigation:</span>
-          <div class="dropdown-content">
-              <a href="index.html" >home</a>
-              <br><a href="about.html">about</a>
-              <br><a href="portfolio.html">portfolio</a>
-            </div>
-        </div>
+      <nav>
+        <ul>
+          <li><a href="index.html" >home</a></li>
+          <li><a href="about.html">about</a></li>
+          <li><a href="portfolio.html">portfolio</a></li>
+        </ul>
       </nav>
-      <br>
   </header>
+
   <main>
-      <img src="/Users/shelby_faulconer/ada/projects/personal-portfolio-site/assets/meish.png" alt="Photo of myself.">
+    <img src="/Users/shelby_faulconer/ada/projects/personal-portfolio-site/assets/meish.png" alt="Photo of myself.">
 
-    <p>
-      <h2>Shelby Rosel Faulconer</h2>
-      <div id="about-me">
-        <h3>&hearts;&hearts;&hearts;<br>
-          <img src="/Users/shelby_faulconer/ada/projects/personal-portfolio-site/assets/biju.gif" alt="gif of Biju from Hamtaro">
-        </h3>
-        My name is Shelby (shell + bee). I study full-stack development and have a rich background in biology, mathematics, and chemistry.
+    <h2>Shelby Rosel Faulconer</h2>
+    <div id="about-me">
+      <h3>&hearts;&hearts;&hearts;</h3>
+      
+      <img src="/Users/shelby_faulconer/ada/projects/personal-portfolio-site/assets/biju.gif" alt="gif of Biju from Hamtaro">
+
+      <p>My name is Shelby (shell + bee). I study full-stack development and have a rich background in biology, mathematics, and chemistry.
         Expressing myself is easiest through visual mediums and writing music.
-        <br>My passions extend deeply and I am a forever student.
-        <h3>
-            <img src="/Users/shelby_faulconer/ada/projects/personal-portfolio-site/assets/theythem.webp" alt="Moving image of license plate with pronouns they/them" width=150px>
-        <br>&hearts;&hearts;&hearts;</h3>
-      </div>
-    </p>
+        My passions extend deeply and I am a forever student.</p>
+      <span>
```

Comment:
Is this span needed for styling? I was wondering if the `img` tag needs to be wrapped or if the `span` could be removed.

Reply:
I tried with and without,  I added the span instead of using a hard bracket in the CSS for this particular image so that it could be constrained to a certain size. I honestly did this kind of lazily instead of looking up the CSS syntax and added the span to keep this image smaller.